.github/workflows/test-e2e-flow.yml

```diff
@@ -0,0 +1,76 @@
+name: Test E2E Flow
+run-name: Test E2E Flow by @${{ github.actor }}
+
+on:
+  workflow_call:
+    inputs:
+      branch:
+        required: true
+        type: string
+      deployment-type:
+        required: true
+        type: string
+      region:
+        required: true
+        type: string
+      serverless:
```

Comment:
Doesn't `type` already covers if it's serverless or not?

Reply:
@romulets, you're right, we can determine that from the deployment-type. I'll refactor and remove the serverless parameter.

Reply:
I've decided to remove `deployment-type` and keep `serverless`, as removing `deployment-type` requires significantly fewer changes.